lib/pure/strutils.nim

```diff
@@ -254,6 +313,34 @@ func toUpperAscii*(s: string): string {.rtl, extern: "nsuToUpperAsciiStr".} =
     doAssert toUpperAscii("FooBar!") == "FOOBAR!"
   toImpl toUpperAscii
 
+func removePunctAscii*(s: string): string {.rtl, extern: "nsuRemovePunctAscii".} =
+  ## returns the a version of `s`, where all Punctuation characters are
+  ## removed. ('!', ',', '.', ':', ';', '?')
+  ## 
+  ## This works on ASCII characters only.
+  runnableExamples:
+    doAssert removePunctAscii("Hello, World!") == "Hello World"
+    doAssert removePunctAscii("No. More. Punctuation!") == "No More Punctuation"
+  let sLen = s.len
+  result = newStringOfCap(sLen)
+  for i in 0..sLen-1:
+    if not (s[i] notin Punctuation):
+      result[i] = s[i]
+
+func removeSpecialAscii*(s: string): string {.rtl, extern: "nsuRemoveSpecialAscii".} =
+  ## returns the a version of `s`, where all Special characters are
+  ## removed. This means all non-alphanumeric characters.
+  ## 
+  ## This works on ASCII characters only.
+  runnableExamples:
+    doAssert removeSpecialAscii("*nuzzle-wuzzles, uwu*") == "nuzzlewuzzles uwu"
```

Comment:
`nim doc -r --lib:lib lib/pure/strutils.nim` fails for me

Reply:
I initually added that as a joke, but it's strange it failed... :/
Well, it's getting removed, either way